browser/components/ContributionsTable.tsx

```diff
@@ -206,44 +207,39 @@ class ContributionsTable extends React.Component<Props, State> {
       </div>
     );
   }
-  getSearchResultText = (arrayList) => {
-    if (arrayList.length !== 0) {
-      return (<h4>Use the search bar and then click the project name to display the result</h4>);
-    }
-    else {
-      return (<h4>No contributions found</h4>);
-    }
-  }
+
   getTable = (arrayList) => {
     if (arrayList.length === 0) {
       return (<h4>No contributions found</h4>);
     } else {
       return this.state.tables.get(this.state.selectedProject);
     }
   }
+
   render() {
-    let count = 0;
-    let newList = this.state.projectNames.filter(name => {
+    const newList = this.state.projectNames.filter(name => {
       return name.toLowerCase().indexOf(this.state.search) > -1;
     });
-    let table;
-    if (this.state.selectedProject === null) {
-      table = this.getSearchResultText(newList);
-    } else {
-      table = this.getTable(newList);
-    };
     return (
+      <div className="container-fluid">
       <div className="row">
-        <div className="col-md-2">
-        <br/>
-          <input type="text" id="projectLISearch" onInput={this.storeSearch} placeholder="Search projects.."/>
-          <ul id="projectUL" className="media-list">
-            {newList.map(name => (<li className="media" key={count++}><a href="#" onClick={() => this.updateSelectedProject(name)}>{name}</a></li>))}
-          </ul>
+          <div className="col-sm-3">
+            <input type="text" id="projectLISearch" className="form-control"
+              onInput={this.storeSearch} placeholder="Search projects" />
+            <div id="projectUL" className="list-group mt-1">
```

Comment:
This had a restricted height so you could scroll the list without moving the page. Will need that back before merge.